content/en/docs/concepts/services-networking/connect-applications-service.md

```diff
@@ -229,45 +229,14 @@ Till now we have only accessed the nginx server from within the cluster. Before
 * An nginx server configured to use the certificates
 * A [secret](/docs/concepts/configuration/secret/) that makes the certificates accessible to pods
 
-You can acquire all these from the [nginx https example](https://github.com/kubernetes/examples/tree/master/staging/https-nginx/). This requires having go and make tools installed. If you don't want to install those, then follow the manual steps later. In short:
```

Comment:
Hi @patrocinio
IMO we should only remove the steps of the Make file which creates confusion. we mention this [nginx https example](https://github.com/kubernetes/examples/tree/master/staging/https-nginx/) so that both options will be there for users. so if someone wants to create this with Makefile then he clicks on this link and follows the steps there.

Reply:
Hi @lavalamp @thockin
PTAL!